recipes/oci/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "oci" %}
+{% set version = "2.88.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oci-{{ version }}.tar.gz
+  sha256: c7c1ad5468417b23349487796a2b6145379c06f829e21ca445b6ae81673058bb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - certifi
+    - cryptography >=3.2.1,<=37.0.2
+    - pyopenssl >=17.5.0,<=22.0.0
+    - python-dateutil >=2.5.3,<3.0.0
+    - pytz >=2016.10
+    - circuitbreaker >=1.3.1,<2.0.0
+
+test:
+  imports:
+    - oci
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://docs.oracle.com/en-us/iaas/tools/python/latest/index.html
+  summary: Oracle Cloud Infrastructure Python SDK
+  license: Apache-2.0
+  license_file: LICENSE.txt
```

Since it was a little painful to find, let's add:
```suggestion
  license_file: LICENSE.txt
  dev_url: https://github.com/oracle/oci-python-sdk
```